lib/services.js

```diff
@@ -224,6 +227,20 @@ export default {
             READ_ACP: [],
         };
 
+        const options = {};
```

Comment:
please add comments in the next 10 or so lines as to why these settings are being set to the options and the omVal